src/components/navigation/right-nav.tsx

```diff
@@ -207,7 +200,8 @@ export class RightNavComponent extends BaseComponent<IProps, IState> {
       } else {
         // TODO: Again, this is for dataflow only. By using the override,
```

Comment:
I don't think this comment is particularly helpful anymore..

Reply:
Good catch -- I tried to remove all such comments but obviously I missed one.

Reply:
Good catch -- I tried to remove all such comments but obviously I missed one.